Clean up README.md and easy.yaml (#133)
Berserker66/MultiWorld-Utilities: README.md

Berserker's Multiworld
======================

A Multiworld implementation for the Legend of Zelda: A Link to the Past Randomizer.  
For setup and instructions there's a [Wiki](https://github.com/Berserker66/MultiWorld-Utilities/wiki).  
Downloads can be found at [Releases](https://github.com/Berserker66/MultiWorld-Utilities/releases), including compiled windows binaries.  

Additions/Changes compared to Bonta's V31
-----------------

Project
 * Available in precompiled form and guided setup for Windows 64Bit on the [Releases](https://github.com/Berserker66/MultiWorld-Utilities/releases) page
 * Compatible with Python 3.7 and 3.8. Forward Checks for Python 4.0 are done

>  * Update modules if they are too old. This prevents crashes when trying to connect among potential other issues

Reviewer: ```suggestion
 * Update modules if they are too old to prevent crashes and other possible issues.
```
Two sentences in a bullet point seems like a lot, I figure that condensing them but retaining the intent is good.

 * Various fixes
 * Overworld Glitches Logic
 * Newer Entrance Randomizer Logic, allowing more potential item and boss locations
 * completely redesigned command interface, with `!help` and `/help`
 * New Goal: local triforce hunt - limits triforce pieces to your own world so it is your own goal to accomplish
 * New Goal: local triforce hunt - Keeps triforce pieces local to your world
 
MultiMystery.py
 * Allows you to generate a Multiworld with individual player mystery weights. Since weights can also be set to 100%, this also allows for individual settings for each player in a regular multiworld.

> Basis is a .yaml file that sets these weights. You can find an [easy.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/easy.yaml) in this project folder to get started.

Reviewer: I forgot to point this out in my review. I would suggest keeping this text as ".yaml" or even ".YAML". The dot isn't redundant in my opinion. This prevents a misinterpretation of the file type as being anything other than a specific file format, even for those systems without typical file extensions.

Author: Address in commit https://github.com/Berserker66/MultiWorld-Utilities/pull/133/commits/dd2694ff1c8f6b097ea4708ed368958c37e7fd63 .

For some reason I kept reading it out loud in my head as dot yaml and it sounded odd but I agree that it does prevent confusion if its explicitly called out as .yaml.

 * Allows a new Mode called "Meta-Mystery", allowing certain mystery settings to apply to all players
   * For example, everyone gets the same but random goal
 
 MultiServer.py
  * Supports automatic port-forwarding, can be enabled in [host.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/host.yaml)
  * Added commands `/hint` and `!hint`. See [host.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/host.yaml) for more information
  * Updates have been made to the following commands:
    * `!players` now displays the number of connected players, expected total player count, and which players are missing

>     * `forfiet` now works when a player is no longer connected

Author: Address in commit https://github.com/Berserker66/MultiWorld-Utilities/pull/133/commits/dd2694ff1c8f6b097ea4708ed368958c37e7fd63

  * Added commands `/hint` and `!hint`. See [host.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/host.yaml) for more information
  * Updates have been made to the following commands:
    * `!players` now displays the number of connected players, expected total player count, and which players are missing
    * `forfiet` now works when a player is no longer connected
    * `/send`, `/hint`, and various other commands now use "fuzzy text matching". It is no longer required to enter a location, player name or item name perfectly
  * Some item groups also exist, so `/hint Bottles` lists all bottle varieties

Mystery.py
 * Defaults to generating a non-race ROM (Bonta's only makes race ROMs at this time)
 * Defaults to generating a non-race ROM (Bonta's only makes race ROMs at this time).
If a race ROM is desired, pass --create-race as argument to it
 * When an error is generated due to a broken .yaml file, it now mentions in the error trace which file, line and character is the culprit
 * Option for progressive items, allowing you to turn them off (see easy.yaml for more info)

>  * Rom-Option for extendedmsu (see easy.yaml for more info)

Author: Per the pull request description. I removed this line as it looks like extendedmsu was removed in this [commit](https://github.com/Berserker66/MultiWorld-Utilities/commit/7f66133d379ad661968f107f76172f152174bbed#diff-28113ab2481a9f388e0c8132ca52acb8)
